example/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 project(capo-example)
+set(cxx_standard cxx_std_20)
```

Comment:
Not needed

Reply:
At least CXX17 is; I'm using if statements with initializer in the sound.cpp demo

Reply:
Oh, then just link to `capo::options` for all compile settings etc.